storage/pebble/registers.go

```diff
@@ -61,7 +69,14 @@ func (s *Registers) Store(
 	height uint64,
 	entries flow.RegisterEntries,
 ) error {
-	//  TODO: add value check once LatestHeight is implemented
+	// value check
+	firstHeight, latestHeight, err := s.getHeightRange()
```

Comment:
We could cache the height range, and check with the cache, because `Store` won't be called concurrently, the cache can be consistent with the database.